src/Extensions/BindExtensions.cs

```diff
@@ -61,15 +61,26 @@ public static T Bind<T>(this HttpRequest request)
                     }
 
                     postedFiles.Add(file);
+                    
+                    if (returnOnFirst)
+                    {
+                        return postedFiles;
+                    }
                 }
             }
 
             return postedFiles;
         }
 
+        public static async Task<IEnumerable<IFormFile>> BindFiles(this HttpRequest request)
+        {
+            return await request.BindFiles(false);
```

Comment:
set the parameter name? BindFiles(returnOnFirst:false)

Reply:
It's a private method with one parameter, really?